### PR TITLE
Install openldap on the admin node? [BSC#1072017]

### PR DESCRIPTION
**Pull request by lproven**
Recreated branch because Oxygen reflowed the whole file again.

Simplified the instructions to just say use a different machine -- not specifying OS.

Wrapped instructions at 80char mark.

### CODE REVIEW

File: xml/deployment_cluster_administration.xml [hunk 1]
```diff
@@ -106,26 +106,26 @@ mail: <email address>
...
-     package:
-    </para>
-    <screen>&prompt.root; <command>zypper</command> <option>install openldap2</option></screen>
-    <para>
-     Then generate the password hash:
+     To generate the password hash, on a separate machine, install the 
+     <package>openldap2</package> package:
     </para>
+    <screen>
+     &prompt.root; <command>zypper</command> <option>install openldap2</option>
```
Comment (cjschroder) [inline xml/deployment_cluster_administration.xml]: Make screen lines flush left, unless you want leading spaces

File: xml/deployment_cluster_administration.xml [hunk 2]
```diff
@@ -106,26 +106,26 @@ mail: <email address>
...
+     Make sure to replace all the parameters in the template above, indicated as &lt;....&gt;.
+     The mail attribute value is used as the login to &dashboard; and &kube;.
     </para>
     <para>
-     Populate your OpenLDAP server with this LDIF file:
+     Transfer the resulting LDIF file to your CaaSP server, and populate your 
+     OpenLDAP server with this file:
     </para>
     <screen>&prompt.root;<command>ldapadd</command> -H ldap://&lt;ADMIN NODE IP&gt;:389 -ZZ -D cn=admin,dc=infra,dc=caasp,dc=local -w &lt;LDAP ADMIN PASSWORD&gt; -f &lt;LDIF FILE&gt;</screen>
     <para>
```
Comment (cjschroder) [inline xml/deployment_cluster_administration.xml]: User-defined variables should use the replaceable tags rather than angle braces.

Comment (cjschroder) [reply xml/deployment_cluster_administration.xml]: also, check the style guide for the correct conventions for example IP addresses and host and domain names. Some of them have entities.

File: xml/deployment_cluster_administration.xml (hunk 1 above)
Comment (lproven) [reply xml/deployment_cluster_administration.xml]: Done.

File: xml/deployment_cluster_administration.xml (hunk 2 above)
Comment (lproven) [reply xml/deployment_cluster_administration.xml]: Good catch. This required removing CDATA tags but I have replaced all the &gt; ... &lt; markup with <replaceable>.

Where the instructions referred to the angle brackets, I have replaced it with 

... indicated <replaceable>like this</replaceable>.

I've used entities for admin node, worker node and the product name. Those are all I could identify.

**Comment by lproven** [comment]
Revised according to Carla's suggestions from Wednesday.

**Comment by cwickert** [comment]
I find it impossible to review the changes given that the whole file was rewritten.

**Comment by lproven** [comment]
No it was not intended. If you can tell me how to  reset back to the previous commit, I will redo the corrections Carla called for without re-wrapping.

**Comment by cwickert** [comment]
Any progress here?

File: xml/deployment_cluster_administration.xml [hunk 3]
```diff
@@ -92,55 +94,56 @@
...
-     Make sure to replace all the parameters in the template above, indicated
-     as &lt;....&gt;. The mail attribute value is used as the login to
+     Make sure to replace all the parameters indicated <replaceable>like this</replaceable>
+     in the template above. The mail attribute value is used as the login to
      &dashboard; and &kube;.
     </para>
     <para>
-     Populate your OpenLDAP server with this LDIF file:
+     Transfer the resulting LDIF file to your &productname; server and populate your
```
Comment (cwickert) [inline xml/deployment_cluster_administration.xml]: For the record: You don't transfer the file, you use the LDAP client tools to connect to the server.

**Comment by cwickert** [comment]
This was dealt with in e8d44f35cc0117be6ad990b3bcdfcb08904b2b05 in the meantime.

**Pull request closed without merging**